tensorflow_probability/python/experimental/mcmc/sample_fold.py

```diff
@@ -18,15 +18,20 @@
 from __future__ import division
 from __future__ import print_function
 
+import warnings
+
 # Dependency imports
 import tensorflow.compat.v2 as tf
-from tensorflow_probability.python.experimental.mcmc import sample
+from tensorflow_probability.python.experimental.mcmc import sample as exp_sample
```

Comment:
We usually call these '_lib', i.e. `sample_lib`

Reply:
gotcha, I called it `exp_sample_lib` just to put emphasis on the fact that it's sample from experimental (the normal mcmc.sample is imported as well)